includes/checkout/template.php

```diff
@@ -770,21 +770,45 @@ function edd_terms_agreement() {
 	if ( edd_get_option( 'show_agree_to_terms', false ) ) {
 		$agree_text  = edd_get_option( 'agree_text', '' );
 		$agree_label = edd_get_option( 'agree_label', __( 'Agree to Terms?', 'easy-digital-downloads' ) );
-		
+
 		ob_start();
 ?>
 		<fieldset id="edd_terms_agreement">
-			<div id="edd_terms" style="display:none;">
+			<div id="edd_terms" class="edd-terms" style="display:none;">
 				<?php
 					do_action( 'edd_before_terms' );
 					echo wpautop( stripslashes( $agree_text ) );
 					do_action( 'edd_after_terms' );
 				?>
 			</div>
-			<div id="edd_show_terms">
+			<div id="edd_show_terms" class="edd-show-terms">
 				<a href="#" class="edd_terms_links"><?php _e( 'Show Terms', 'easy-digital-downloads' ); ?></a>
 				<a href="#" class="edd_terms_links" style="display:none;"><?php _e( 'Hide Terms', 'easy-digital-downloads' ); ?></a>
 			</div>
+
+			<?php if ( ! edd_get_option( 'show_agree_to_privacy_policy', false ) && edd_get_option( 'show_to_privacy_policy_on_checkout', false ) ) : ?>
+				<?php
+				$agree_page      = edd_get_option( 'privacy_agree_page', get_option( 'page_for_privacy_policy' ) );
+				$agree_label     = edd_get_option( 'privacy_agree_label', __( 'Agree to Terms?', 'easy-digital-downloads' ) );
+				$agreement_text  = get_post_field( 'post_content', $agree_page );
+				?>
+
+				<?php if ( ! empty( $agreement_text ) ) : ?>
+					<div id="edd-privacy-policy" class="edd-terms" style="display:none;">
+						<?php
+						do_action( 'edd_before_privacy_policy' );
+						echo wpautop( stripslashes( $agreement_text ) );
```

Comment:
This should either call `do_shortcode` or should run the `the_content` filter to ensure shortcodes on the page get parsed.

Reply:
:+1:

Reply:
@cklosowski I'm not seeing shortcodes working here. Am I understanding correctly that they are supposed to now?